script.js

```diff
@@ -0,0 +1,19 @@
+
+var arr = new Array();
+function myFunction(name,breed,fee){
+  alert("Name:"+name+"\nBreed:"+breed+"\nAdoption Fee:"+fee);
+}
+
+function getSum(total,num) {
+  return total + Math.round(num);
+}
+
+function adoptFunction(item){
+  var x = document.getElementById("price").childNodes[0].nodeValue;
+  arr.push(x);
+
+  for (var i = 0; i <= arr.length; i++) {
+    total = +arr[i];
```

Comment:
Using a variable 'total' without ever defining it is bad practice. Should use let. using it in this way creates a global.